src/java.base/share/man/java.md

```diff
@@ -2502,9 +2502,9 @@ Java HotSpot VM.
 `-XX:MaxGCPauseMillis=`*time*
 :   Sets a target for the maximum GC pause time (in milliseconds). This is a
     soft goal, and the JVM will make its best effort to achieve it. The
-    specified value doesn't adapt to your heap size. By default, for G1 the
-    maximum pause time target is 200 milliseconds. The other generational
-    collectors do not use a pause time goal by default.
+    specified value doesn't adapt to your heap size. G1's default for the
+    maximum pause time target is 200 milliseconds and only G1 currently supports
+    a maximum GC pause time target.
```

Comment:
```suggestion
    specified value doesn't adapt to your heap size.
    Currently only G1 supports a maximum GC pause time target. The default value
    is 200 milliseconds.
```

Reason for this suggestion:
* it seems more important that only G1 supports this value than the default value, so put this first
* these are two separate statements, so put them into two separate sentences. The default value is also independent of the collector.

Reply:
> The specified value doesn't adapt to your heap size.

Should we consider putting this last as that seems like a less important implementation detail compared to which GC support it and the default value?

Reply:
Actually we can leave this sentence completely out.